data/hai/hai reveal 3 early.txt

```diff
@@ -0,0 +1,435 @@
+	landing
+	source "Hai-home"
+	stopover "Makerplace"
+	destination "Mountaintop"
+	clearance
+	to offer
+		has "Hai Leaks Response 6: done"
+	on offer
+		conversation
+			`When you return to Hai-home you are directed now to a government hangar where you are met by Xilin and Sayari, both looking very relieved.`
+			`	"I am very glad to see you returned Captain <last>, Sayari said you were away on an important errand but I've been hoping for a day now you would appear, as I think you might be the solution to the current impasse."`
+			`	"You see," says Sayari, "in the light of the recent civil war your species has had, some of the Elder's are less than keen with allowing a partisan observer to see the inner workings of a station in its earliest phases, especially when we haven't built one in so, so long."`
+			`	Now that she mentions it, you don't think you've seen a Hai station at all.`
+			`	"However, therein lies a problem," says Xilin.`
+			`	"The Republic will not allow the Hai to build a station this side of the wormhole without some oversight by someone we trust. This has now been a point of contention for some days, and multiple suggestions have been rejected."`
+			`	Sayari chitters, "Honestly it's all very silly. Lots of old men who are meant to be friends concerned that the other might have been lying to them."`
+			`	In an oddly comical gesture they both shrug at the same time.`
+			`	"Anyway, thanks to a little Freedom of Information request to the Republic, I have been looking at things and I think I have the solution."`
+			choice
+				`	"Is it me?"`
+			`Sayari winks at you, "Catch on quick don't you."`
+			`	"The actions you're recorded as having taken show a certain conscience for the good of your species as a whole, despite your affiliations. I have convinced the other diplomats and the Elders that you might be a viable solution to this impasse. The human diplomats have already agreed, but the Elders wish to speak with you first."`
+			choice
+				`	"Who are these Elders?"`
+				`	"Of course, that's natural."`
+					goto main
+			`	Sayari explains, "The Hai are ruled by an elected 'Council of Elders', they are not responsible for every administrative decision that occurs but they are the highest authority of our government. Their decisions are final, at least for the term of an election."`
+			label main
+			`	"If you're willing, there will be a council meeting in an hour, I can put you first on the agenda. Will you do this?" asks Sayari.`
+			choice
+				`	"I suppose I must."`
+					goto council
+				`	"Someone better be paying me for this."`
+					goto council
+				`	"Of course, let's do it."`
+					goto council
+				`	"Maybe in a bit, I should do something else first."`
+					defer
+			label council
+			`Following Sayari you pass beyond a security checkpoint where Xilin is prevented from following you. Sayari speaks with another Hai doing administrative work for a moment before gesturing you to a private antechamber.`
```

Comment:
Might be better said as 'gesturing you towards a private antechamber' or 'directing you to a private antechamber'